src/modules/gauging-stations/jobs/sync-gauging-stations.js

```diff
@@ -0,0 +1,105 @@
+// Syncs gauging stations from CSV on S3
+
+'use strict';
+
+// Dependencies
+const moment = require('moment');
+const { logger } = require('../../../logger');
+const s3Connector = require('../../../lib/services/s3');
+
+// Constants
+const JOB_NAME = 'gauging-stations.sync-from-csv';
+const csvKey = 'gauging-stations/gauging-stations.csv';
+
+// Handy stuff
+const config = require('../../../../config');
+const helpers = require('../helpers');
+const applicationState = require('../../../lib/services/application-state');
+
+// Gauging Stations Repo
+const gaugingStationsRepo = require('../../../lib/connectors/repos/gauging-stations');
+const gaugingStationMapper = require('../../../lib/mappers/gauging-station');
+
+const createMessage = () => ([
+  JOB_NAME,
+  {},
+  {
+    jobId: `${JOB_NAME}.${moment().format('YYYYMMDD')}`,
+    repeat: {
+      every: config.import.gaugingStationsSyncFrequencyInMS
+    }
+  }
+]);
+
+const handler = async () => {
+  logger.info(`${JOB_NAME}: Job has started`);
+
+  // Get the Body and the ETag of the S3 object
```

Comment:
For readability I'd maybe suggest splitting out this bit of logic that fetches and parses the CSV data into a separate function, returning array or null if etag matches app state.